catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/src/main/java/org/codice/ddf/spatial/ogc/wfs/v1_1_0/catalog/converter/impl/GenericFeatureConverterWfs11.java

```diff
@@ -0,0 +1,246 @@
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wfs.v1_1_0.catalog.converter.impl;
+
+import com.thoughtworks.xstream.converters.MarshallingContext;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
+import com.vividsolutions.jts.geom.Geometry;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.AttributeType.AttributeFormat;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+import java.io.Serializable;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Base64;
+import java.util.Date;
+import java.util.Set;
+import java.util.TreeSet;
+import javax.xml.namespace.QName;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.time.DateFormatUtils;
+import org.codice.ddf.spatial.ogc.catalog.common.converter.XmlNode;
+import org.codice.ddf.spatial.ogc.wfs.catalog.common.AttributeDescriptorComparator;
+import org.codice.ddf.spatial.ogc.wfs.catalog.common.WfsConstants;
+import org.codice.ddf.spatial.ogc.wfs.catalog.common.WfsQnameBuilder;
+import org.codice.ddf.spatial.ogc.wfs.catalog.converter.impl.EnhancedStaxWriter;
+import org.codice.ddf.spatial.ogc.wfs.catalog.mapper.MetacardMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class works in conjunction with XStream to convert a {@link Metacard} to XML according to
+ * the GML 3.1.1 spec. It will also convert respective XML into a Metacard.
+ */
+public class GenericFeatureConverterWfs11 extends AbstractFeatureConverterWfs11 {
+
+  private static final String ID = "id";
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(GenericFeatureConverterWfs11.class);
+
+  private String sourceId = null;
+
+  public GenericFeatureConverterWfs11(String srs) {
+    super(srs);
+  }
+
+  public GenericFeatureConverterWfs11(MetacardMapper metacardMapper) {
+    super(metacardMapper);
+  }
+
+  /**
+   * Method to determine if this converter knows how to convert the specified Class.
+   *
+   * @param clazz the class to check
+   */
+  @Override
+  public boolean canConvert(Class clazz) {
+    return Metacard.class.isAssignableFrom(clazz);
+  }
+
+  /**
+   * This method will convert a {@link Metacard} instance into xml that will validate against the
+   * GML 2.1.2 AbstractFeatureType.
+   *
+   * @param value the {@link Metacard} to convert
+   * @param writer the stream writer responsible for writing this xml doc
+   * @param context a reference back to the Xstream marshalling context. Allows you to call
+   *     "convertAnother" which will lookup other registered converters.
+   */
+  @SuppressWarnings("ConstantConditions")
+  @Override
+  public void marshal(Object value, HierarchicalStreamWriter writer, MarshallingContext context) {
```

Comment:
❗️ Again, I don't think we need the marshall implementation

Reply:
change to throw an UnsupportedOperationException